### ISSUE TITLE
no alt text

### ISSUE THREAD

**Issue by SalocinHB** [pixelfed/mobile-app#29]
Alt text for images entered in the app seems to be lost on upload.

**Comment by osresearch** [comment]
This is still a problem with 1.0.0(4).  No alt text is added via either the `Aa` button in the image selector nor via the `Advanced Settings` - `Write Alt Text`.

**Comment by SalocinHB** [comment]
Yes, still broken.

**Comment by osresearch** [comment]
Hurrah, this was fixed in 1.0.0(5) released today.
